packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/DestinationNetworkBox.tsx

```diff
@@ -138,7 +146,7 @@ function BalancesContainer() {
       className="rounded px-3 text-white [&>*+*]:border-t [&>*+*]:border-gray-600"
       style={{ backgroundColor: '#00000050' }}
     >
-      {isCctpTransfer && isDepositMode && (
+      {isCctpTransfer && transferMode === 'deposit' && (
```

Comment:
what about teleport?

Reply:
teleport mode will always be false for `isCctpTransfer` so we don't have to check for it